ts/components/PreferencesList.tsx

```diff
@@ -0,0 +1,22 @@
+import { List, View } from "native-base";
```

Comment:
add block comment describing the purpose of this component

Reply:
Fixed.